### PR TITLE
secret policies + terminator class

### PR DESCRIPTION
**Pull request by abikouo**
Required for pull request: https://github.com/ansible-collections/amazon.aws/pull/351

### CODE REVIEW

**Comment by jillr** [comment]
Thanks @abikouo!  You have absolutely the right idea here, but we've needed to do larger a refactor for a while since security-services.yaml wasn't the only policy file that needed work.  I've done that in #150.  It did involved a new file, but once I sat down and looked at what things could logically move together I ended up creating an application-security file instead.  Once that lands, I checked and there will be enough space in security-services.yaml for these permissions.

The permissions listed here look good to me.  We'll also need a terminator class though, similar to this https://github.com/mattclay/aws-terminator/blob/master/aws/terminator/security_services.py#L47:L72

**Comment by abikouo** [comment]
> Thanks @abikouo! You have absolutely the right idea here, but we've needed to do larger a refactor for a while since security-services.yaml wasn't the only policy file that needed work. I've done that in #150. It did involved a new file, but once I sat down and looked at what things could logically move together I ended up creating an application-security file instead. Once that lands, I checked and there will be enough space in security-services.yaml for these permissions.
> 
> The permissions listed here look good to me. We'll also need a terminator class though, similar to this https://github.com/mattclay/aws-terminator/blob/master/aws/terminator/security_services.py#L47:L72

@jillr thanks for your feedback on this PR. 
I will wait for #150 to be merged then realign on this and also create the terminator class

**Comment by jillr** [comment]
@abikouo 150 has been merged, thanks!

**Comment by abikouo** [comment]
@jillr could you please review this one?

File: aws/policy/security-services.yaml
Comment (jillr) [inline aws/policy/security-services.yaml:102]: Both Describe and List need to be in the above group (`AllowGlobalUnrestrictedResourceActionsWhichIncurNoFees`, operating on `Resource: "*"`)

File: aws/policy/security-services.yaml
Comment (jillr) [inline aws/policy/security-services.yaml:108]:
```suggestion
      - secretsmanager:UntagResource
      - secretsmanager:TagResource
```

**Review by jillr — COMMENTED** [review]
With the noted comments I was able to get your tests in #351 to pass, and the terminator class works.  Slack me when you're ready for a review and I'll look asap!  Thanks for your work on this, sorry for the delays.

**Review by jillr — APPROVED** [review]
Passing with 351, thanks very much for your work on this!

**Pull request merged**
